src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,19 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
+    private String name;
+    private Engine engine;
+    private String color;
+    private int doorCount;
+    private int issueYear;
 
+    public Car(String name, Engine engine, String color, int doorCount, int issueYear) {
+    }
+
+    @Override
+    public Car clone() throws CloneNotSupportedException {
+        Engine cloneEngine = engine;
```

Comment:
will you get the copy of of engine or the same engine?

Reply:
Fixed. Done